test/integration/RSR-integration.test.ts

```diff
@@ -4,41 +4,46 @@ import { expect } from 'chai'
 import hre, { ethers } from 'hardhat'
 
 import { ForkSpell } from '../../typechain/ForkSpell'
-import { MultiSigWalletWithDailyLimit } from '../../typechain/MultiSigWalletWithDailyLimit'
 import { ReserveRightsToken } from '../../typechain/ReserveRightsToken'
 import { RSR } from '../../typechain/RSR'
 import { SiphonSpell } from '../../typechain/SiphonSpell'
-import { SlowWallet } from '../../typechain/SlowWallet'
 import { UPGRADE_SIPHONS } from './../../scripts/deployment/siphon_config'
 import { impersonate } from './utils/accounts'
-import { ONE, ZERO } from '../../common/numbers'
+import { ONE, ZERO, bn } from '../../common/numbers'
 import { WEIGHT_ONE, ZERO_ADDRESS } from '../common'
 
+// TODO: Add more siphon test cases when the contract is deployed
+// For the mainnet fork, only test the first 5 addresses with balances using different weight values
+const mockWeights = [bn('0'), bn('0.5e18'), WEIGHT_ONE, bn('0.2e18'), bn('0.75e18')]
```

Comment:
Why is the reason of doing this and not testing all the real siphons? because the weights all match? Anycase I would remove the TODO for a Note: (to complete the idea of no todo's)

Something like 'Note: We  can add more siphon tests once the contract is deployed`

Reply:
Yep, they all match atm and it would be like doing the same test N times